botocore/httpsession.py

```diff
@@ -317,7 +318,7 @@ def _get_proxy_manager(self, proxy_url):
             proxy_headers = self._proxy_config.proxy_headers_for(proxy_url)
             proxy_manager_kwargs = self._get_pool_manager_kwargs(
                 proxy_headers=proxy_headers)
-            proxy_manager_kwargs.update(**self._proxies_kwargs)
+            proxy_manager_kwargs.update(**self._proxies_kwargs(proxy_url))
```

Comment:
If we're going to start handing down arguments, let's decouple this a bit. It is likely better to keep `_proxies_kwargs` as our static settings and then have an additional step that creates and adds the `proxy_ssl_context` in a separate step.

We don't want to have to require access to every possible SSLContext configuration option when we're pulling static args such as line 389.

Reply:
Yes, makes sense. I can separate them out